build.sh

```diff
@@ -100,3 +119,11 @@ if [ $? -ne 0 ]; then
   echo 'Problem encountered building deployment resources'
   exit
 fi
+
+#
+# If running in development mode, activate watch mode
+#
+if [ "$DEVELOPMENT" == 'true' ]; then
```

Comment:
I'm not a fan of the spa being started as part of the `build` script. Maybe there should be another script that starts the spa?